account_invoice_payment_mode_note_template/models/account_payment_mode.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from markupsafe import Markup
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountPaymentMode(models.Model):
+    _inherit = "account.payment.mode"
+
+    note = fields.Html(translate=True)
+
+    @api.constrains("note")
+    def _check_note(self):
+        invoice = self.env["account.move"].search([], limit=1)
+        for rec in self:
+            rec.render_note(invoice, raise_error=True)
+
+    def render_note(self, invoice, raise_error=False):
+        self.ensure_one()
+        if not self.note:
+            return ""
+        try:
+            result = self.env["mail.template"].render_template(
```

Comment:
`mail.template` object has no attribute `render_template`. Did you mean: `_render_template`?